gluten-flink/runtime/src/main/java/org/apache/gluten/util/LogicalTypeConverter.java

```diff
@@ -58,6 +59,9 @@ public static Type toVLType(LogicalType logicalType) {
             return new io.github.zhztheplayer.velox4j.type.DecimalType(
                     decimalType.getPrecision(),
                     decimalType.getScale());
+        } else if (logicalType instanceof DayTimeIntervalType) {
+            // TODO: it seems interval now can be used as bigint for nexmark.
+            return new io.github.zhztheplayer.velox4j.type.BigIntType();
```

Comment:
Can we use `IntervalDayTimeType()`? IntervalDayTimeType is implemented based on BigIntType in Velox, but they are logically different.

Reply:
IntervalDayTimeType can not be cast to BigIntType now in velox. Change it to BigIntType temporary, I will assgin this issuse to others to fix it.